docs/developer.rst

```diff
@@ -65,6 +65,47 @@ To build ``jaxlib`` without CUDA GPU support (CPU only), drop the ``--enable_cud
   python build/build.py
   pip install -e build  # installs jaxlib (includes XLA)
 
+
+Additional Notes for Building ``jaxlib`` from source on Windows
+...............................................................
+
+On Windows, following `Install Visual Studio <https://docs.microsoft.com/en-us/visualstudio/install/install-visual-studio?view=vs-2019>`_
+to setup latest C++ toolchain. If you need to build with CUDA enabled, follow
+`CUDA Installation Guide <https://docs.nvidia.com/cuda/cuda-installation-guide-microsoft-windows/index.html>`_
+to setup CUDA environment.
+
+Install `bazelisk <https://github.com/bazelbuild/bazelisk/releases/latest>`_.
```

Comment:
I actually suspect this step is optional now the `build.py` script will download bazel for you on Windows.